models.py

```diff
@@ -107,3 +107,52 @@ class CCRecruitment(BaseModel):
         str_strip_whitespace=True,
         validate_assignment=True,
     )
+
+
+class Base64Validator(BaseModel):
+    base64_bytes: Optional[Base64Bytes] = None
+    base64_str: Base64Str
+
+
+class StorageFile(BaseModel):
+    id: PyObjectId = Field(default_factory=PyObjectId, alias="_id")
+    title: str = ""
+    data: Base64Str | None = None
+    filetype: str = "pdf"
+    modified_time: str = ""
+    created_time: str = ""
+
+    @field_validator("data")
+    @classmethod
+    def validate_filedata(cls, value):
```

Comment:
When data is already of type base64str, then what is the need of this validation, which is doing the same thing